plugins/exporter.koplugin/clip.lua

```diff
@@ -286,7 +286,11 @@ function MyClipping:parseHighlight(highlights, bookmarks, book)
             end
         end
     end
-    table.sort(book, function(v1, v2) return v1[1].page < v2[1].page end)
+    local bookmark_timestamps = {}
+    for i = #bookmarks, 1, -1 do
```

Comment:
Can we cycle ascending and change the sign in the sorting function?

Reply:
Done.